gethoney/crud.py

```diff
@@ -1,4 +1,70 @@
 import sqlite3
 
-conn = sqlite3.connect("../data/gethoney.db", check_same_thread=False)
-curr = conn.cursor()
+import requests
+
+from gethoney.models import Honeypot
+
+
+class Database:
+    def __init__(self, db_path: str):
+        self.conn = sqlite3.connect(db_path, check_same_thread=False)
+        self.curr = self.conn.cursor()
+        self.curr.execute(
+            """CREATE TABLE IF NOT EXISTS honeypots
+            (id INTEGER PRIMARY KEY AUTOINCREMENT, name TEXT, url TEXT, description TEXT)"""
+        )
+        self.curr.execute(
+            """CREATE TABLE IF NOT EXISTS logs
+            (id INTEGER PRIMARY KEY AUTOINCREMENT,
+            name TEXT,
+            honeypot_id INTEGER,
+            FOREIGN KEY (honeypot_id) REFERENCES honeypots (id))"""
+        )
+
+    def create_honeypot(self, honeypot: Honeypot):
+        self.curr.execute(
+            """INSERT INTO honeypots
+            (name, url, description)
+            VALUES (?, ?, ?) """,
+            (honeypot.name, honeypot.url, honeypot.description),
+        )
+
+        self.conn.commit()
+
+    def list_honeypots(self):
+        data = self.curr.execute("SELECT * FROM honeypots").fetchall()
+        return data
+
+    def retrieve_logs(self, honeypot: Honeypot):
+        fetch_url_id = self.curr.execute("SELECT url, id FROM honeypots WHERE name == ?", (honeypot.name,)).fetchall()
+        url = fetch_url_id[0][0]
+        id = fetch_url_id[0][1]
+
+        request = requests.get(url)
+        data = request.json()
+
+        self.curr.executemany(
+            """INSERT INTO logs
+            (name, honeypot_id)
+            VALUES (?, ?)""",
+            [(item["name"], id) for item in data],
+        )
+        self.conn.commit()
+
+    def update_honeypot(self, honeypot: Honeypot, _id: int):
```

Comment:
So typically, using an underscore to prefix a variable denotes it as a private variable. In this case, assuming you're doing this so you don't override the built in id function, the you would use a trailing underscore, so `id_`

https://stackoverflow.com/questions/77552/id-is-a-bad-variable-name-in-python 
^ more reading, you can look at the PEP8 style guide for more info